src/running-policies.md

```diff
@@ -0,0 +1,230 @@
+project](https://github.com/chimera-kube/chimera-controller).
+
+With the `chimera-controller` project you will be able to manage
+admission policies in a very easy, safe and straightforward way.
+
+## Deploy the `chimera-controller`
+
+First of all, you will need an operative Kubernetes cluster, with a
+`kubectl` context pointing to it.
+
+1. Deploy `cert-manager`:
+
+    ```console
+    $ kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
+    $ kubectl wait --for=condition=Available deployment --timeout=2m -n cert-manager --all
+    ```
+
+1. Deploy the `chimera-controller` after `cert-manager` is ready:
+
+    ```console
+    $ kubectl apply -f https://raw.githubusercontent.com/chimera-kube/chimera-controller/v0.0.1/config/generated/all.yaml
+    $ kubectl wait --for=condition=Available deployment --timeout=2m -n chimera-controller-system --all
+    ```
+
+At this point, the `chimera-controller` is monitoring custom resources
+of group/version `chimera.suse.com/v1alpha1` and kind
+`AdmissionPolicy`.
+
+## `AdmissionPolicy` resource
+
+The `AdmissionPolicy` resource is the core of the `chimera-controller`
+project. This resource is how you control the admission policies that
+run in your cluster.
+
+`AdmissionPolicy` resources can only be created and deleted at this
+time, they cannot be edited.
+
+> **NOTE:** Admission policies are registered with a `*` webhook
+> `scope`, which means that registered webhooks will be forwarded all
+> requests matching the given `resources` and `operations` -- either
+> namespaced (in any namespace), or cluster-wide resources.
+
+> **NOTE:** The `AdmissionPolicy` resource is cluster-wide. There are
+> plans to also provide a namespaced version that will only impact
+> registered namespaced resources on its own namespace.
+
+```yaml
+apiVersion: chimera.suse.com/v1alpha1
+kind: AdmissionPolicy
+metadata:
+  name: admissionpolicy-sample
+spec:
+  module: registry://ghcr.io/chimera-kube/policies/pod-privileged:v0.0.1
+  resources:
+  - pods
+  operations:
+  - CREATE
+```
+
+* `module`: this is the location of the module, several schemas are
+  supported.
+  * `registry`: an [OCI
+    artifacts](https://github.com/opencontainers/artifacts) compliant
+    registry.
+  * `http`, `https`: download from a regular HTTP server.
+  * `file`: load the module from the local filesystem. This is not
+    recommended, as the admission server will run in a container.
+
+* `resources`: what type of resources should be forwarded to this
+  admission policy by the API server for evaluation.
+
+* `operations`: what operations for the previously given types should
+  be forwarded to this admission policy by the API server for
+  evaluation.
+
+* `env` (optional): an arbitrary map that will be exposed as
+  environment variables to the Wasm module. This allows you to provide
+  static configuration to the module at definition time that will be
+  exposed to the module at runtime.
```

Comment:
We will have to update that part of the docs once we have the controller handle waPC policies.